demo/mobile/package.json

```diff
@@ -1,64 +1,96 @@
-    "grunt-newer": "1.1.2",
-    "grunt-rev": "0.1.0",
-    "grunt-sass": "1.1.0",
-    "load-grunt-tasks": "3.4.1",
-    "time-grunt": "1.3.0",
-    "uglifyify": "~3.0.4"
-  }
-}
+    "name": "@raincatcher/demo-mobile",
+    "version": "1.0.0",
+    "description": "Example demo mobile application",
+    "engines": {
+        "node": ">=4.4 <=6.10"
+    },
+    "scripts": {
+        "start": "grunt",
+        "test": "grunt eslint"
+    },
+    "keywords": [
+        "wfm"
+    ],
+    "author": "feedhenry-raincatcher@redhat.com",
+    "license": "MIT",
+    "dependencies": {
+        "@raincatcher/datasync-client": "1.0.0",
+        "@raincatcher/demo-auth-keycloak": "1.0.0",
+        "@raincatcher/demo-auth-passport": "1.0.0",
+        "@raincatcher/demo-sync": "1.0.0",
+        "@raincatcher/dialog": "1.0.0",
+        "@raincatcher/logger": "1.0.0",
+        "@raincatcher/vehicle-inspection": "1.0.0",
+        "@raincatcher/wfm": "1.0.0",
+        "@raincatcher/workflow-angular": "1.0.0",
+        "@raincatcher/workorder-angular": "1.0.0",
+        "angular": "1.6.4",
+        "angular-animate": "1.6.4",
+        "angular-aria": "1.6.4",
+        "angular-material": "1.1.4",
+        "angular-ui-router": "0.4.2",
+        "async": "1.5.2",
+        "bluebird": "^3.5.0",
+        "cordova-android": "^6.2.3",
+        "debug": "^2.6.3",
+        "fh-js-sdk": "2.18.6",
+        "grunt-cli": "^1.2.0",
+        "keycloak-js": "^3.2.1",
+        "lodash": "4.17.4",
+        "moment": "~2.18.1",
+        "q": "1.5.0",
+        "rx": "~4.1.0",
+        "shortid": "~2.2.6",
+        "underscore": "~1.8.3",
+        "cordova-ios": "~4.2.0",
+        "cordova-plugin-camera": "~2.3.0",
+        "cordova-plugin-console": "~1.0.3",
+        "cordova-plugin-device": "~1.1.2",
+        "cordova-plugin-file": "~4.2.0",
+        "cordova-plugin-file-transfer": "~1.5.1",
+        "cordova-plugin-geolocation": "~2.4.0",
+        "cordova-plugin-inappbrowser": "1.6.1",
+        "cordova-plugin-network-information": "~1.2.1",
+        "cordova-plugin-splashscreen": "~4.0.0",
+        "cordova-plugin-whitelist": "~1.2.2",
+        "phonegap-plugin-barcodescanner": "~6.0.0"
+    },
+    "devDependencies": {
+        "browserify": "13.3.0",
+        "browserify-ngannotate": "~2.0.0",
+        "browserify-shim": "3.8.12",
+        "grunt": "0.4.5",
+        "grunt-browserify": "5.0.0",
+        "grunt-contrib-clean": "1.0.0",
+        "grunt-contrib-connect": "1.0.1",
+        "grunt-contrib-copy": "1.0.0",
+        "grunt-contrib-watch": "1.0.0",
+        "grunt-eslint": "~18.1.0",
+        "grunt-newer": "1.1.2",
+        "grunt-rev": "0.1.0",
+        "grunt-sass": "1.1.0",
+        "load-grunt-tasks": "3.4.1",
+        "time-grunt": "1.3.0",
+        "uglifyify": "~3.0.4"
+    },
+    "cordova": {
+        "platforms": [
+            "android",
+            "ios"
+        ],
+        "plugins": {
+            "cordova-plugin-camera": {},
```

Comment:
Note - this plugins aren't used anymore but we going to need them soon so decided to leave them for the moment.